solr/core/src/test/org/apache/solr/update/PeerSyncTest.java

```diff
@@ -216,23 +218,27 @@ public void test() throws Exception {
     add(client0, seenLeader, sdoc("id", Integer.toString((int) v), "_version_", v));
     docsAdded.add(4000);
     int toAdd = numVersions + 10;
-    for (int i = 0; i < toAdd; i++) {
-      add(
-          client0,
-          seenLeader,
-          sdoc("id", Integer.toString((int) v + i + 1), "_version_", v + i + 1));
-      add(
-          client1,
-          seenLeader,
-          sdoc("id", Integer.toString((int) v + i + 1), "_version_", v + i + 1));
-      docsAdded.add((int) v + i + 1);
-    }
+    int offset = (int) v;
+    add(
+        client0,
+        seenLeader,
+        IntStream.range(0, toAdd)
+            .map(i -> i + offset + 1)
+            .mapToObj(i -> sdoc("id", Integer.toString(i), "_version_", i))
+            .collect(Collectors.toList()));
```

Comment:
Wondering if the logic could be simplified by moving the +offset and +1 into the loop/range?
```suggestion
        IntStream.rangeClosed(1 + offset, toAdd + offset)
            .mapToObj(i -> sdoc("id", Integer.toString(i), "_version_", i))
            .collect(Collectors.toList()));
```